pythoninput.md

```diff
@@ -0,0 +1,64 @@
+Two main methods for taking in input with Python 3:
+***
+
+```python
+input()
+```
+
+Returns line read from input after being converted to a string
+
+Optionally can be provided an argument that is printed to standard ouput
+
+``` python
+>>> a = input("Enter a number\n")
+Enter a number
+5
+>>> print(a)
+'5'
+```
+
+If input is desired in some type other than string, you can convert it like so:
+
+```python
+a = int(input())
+b = float(input())
+```
+
+[input() Documentation](https://docs.python.org/3/library/functions.html#input)
+
+In some cases it may be useful to separate the line of input based on a delimiter 
+such as ' ' or ','
+
+In Python we can do that with:
+
+```python
+split()
```

Comment:
Would it be better to move the split section to the end of the document? It would be useful for both input-reading API.